.github/workflows/check_links.yml

```diff
@@ -31,14 +31,42 @@ jobs:
           key: cache-lychee-${{ github.sha }}
           restore-keys: cache-lychee-
 
-      # Extract and check URLs directly from token files
+      # Extract and check URLs directly from token files (checking dumped file did not work)
       - name: Check token art URLs
         uses: lycheeverse/lychee-action@v1
         with:
-          args: '${{env.lychee_args}} -- tokens.xml challenge_tokens.xml'
```

Comment:
is this intentional?

Reply:
Yes, we no longer have a separate challenge_tokens.xml file.
See #225